src/contrib/cluster/Akka.Cluster.Tools.Tests/PublishSubscribe/DistributedPubSubDeadLetterSpec.cs

```diff
@@ -37,6 +37,16 @@ public async Task DistributedPubSubMediator_should_send_specialized_dead_letter_
             var mediator = DistributedPubSub.Get(Sys).Mediator;
             var testMessage = "test-message";
 
+            // Ensure DeadLetterListener is active by retrying until it logs a warmup message
+            await AwaitAssertAsync(async () =>
```

Comment:
Comment explains it all, but the issue here was the test executing so quickly that the `DeadLetterListener` couldn't complete its `DeadLetter` subscription to the `EventStream`

Reply:
So now we `AwaitAssert` that the subscription is intact before proceeding with the real test